docs/src/developer/install.md

```diff
@@ -1,36 +1,118 @@
-# Installing miniSASS Django-CMS project
+# Setting up a Local Development Environment
 
-This is currently a basic Django-CMS site. No extra Django applications have been added yet. 
-To install the pre-requisites needed for this site to run, do the following:
+## Prerequisites
 
-## Docker Deployments
+- Any Linux distribution for the operating system
+- Docker installed on the local machine
+- Visual Studio Code for the editor (or any preferred)
 
-* Clone the repository
+## Installation Steps
 
+1. **Open the terminal.**
+
+2. **Create a directory:**
+    ```bash
+    mkdir name_of_directory
+    ```
+
+3. **Clone the minisass repository:**
+    ```bash
+    git clone https://github.com/kartoza/miniSASS.git
+    ```
+
+4. **Navigate into the repo directory:**
+    ```bash
+    cd miniSASS
+    ```
+
+5. **Open the directory with VSCode (assuming you have VSCode installed) or open the repository directory in any editor of your choice.**
+
+6. **Optional: Adjust the build context of the S3 mount.**
```

Comment:
What is the implication of actually fixing this in the repo from https://github.com/kartoza/miniSASS/blob/main/deployment/docker-s3-bucket/Dockerfile#L27C1-L27C65 to become
```
ADD ./start-script.sh /start-script.sh
```

Reply:
@NyakudyaA we can remove it if its not needed?